### PR TITLE
add a couple tests

### PR DESCRIPTION
**Pull request by lucask42**

### CODE REVIEW

File: scripts/utils.js [hunk 1]
```diff
@@ -35,16 +35,30 @@ const validAruments = args => {
...
-    console.log(
-      'Unknown command - must be one of template:create, template:delete. e.g. template:create'
-    );
+    if (process.env.NODE_ENV !== 'test') {
+      console.log(
+        'Unknown command - must be one of template:create, template:delete. e.g. template:create'
+      );
+    }
     process.exit(0);
+    return {};
```
Comment (johnuphoff) [inline scripts/utils.js]: I'm not sure this will return...

**Comment by lucask42** [comment]
Yeah, I'll take that out.  My mock 'process.exit' allows the code to
continue

**Comment by lucask42** [comment]
This all stemmed from the fact that `process.exit()` interrupts the testing.  In order to prevent the interruption I had created a mock function that did a noop.  I explored options to end the test early, force the test to pass,  or throw from within the mocked function but those strategies didn't seem to be viable.

In this change I just I removed the `process.exit()` and replaced it with `return {};`

This makes sense because `validAruments()` should return whatever the validArgs are and if there are no valid args or if the command is missing then `validAruments()` should just return an empty object.  This instead of returning a valid args OR killing the process.

Also the `process.exit()` in `validAruments()` is redundant.   When `template.js` gets arguments that have invalid name or invalid type it will go ahead and `process.exit()` and if it gets an unrecognized command it will just noop.

This kind of kicks the can down the road as there may be some more changes to write unit tests for `template.js` that can handle the `process.exit()`s there.

**Comment by johnuphoff** [comment]
Cool yeah thanks - you're right that validArguments should return an empty object there.

**Pull request merged**